CHANGES.md

```diff
@@ -3,6 +3,7 @@ Unreleased
 
 * atdgen: Add option `-j-gen-modules` to generate JSON generic submodules (#420)
 * atd-parser: improve (syntax) error messages (#426)
+* atdts: support <ocaml from...> annotation
```

Comment:
This should be `<ocaml ts=...>`. I would add a slightly longer note explaining what the feature is about.

Reply:
`<ts from=..>` ?